app/dashboard/views.py

```diff
@@ -1141,6 +1151,187 @@ def get_quickstart_video(request):
     return TemplateResponse(request, 'quickstart_video.html', context)
 
 
+@vary_on_cookie
+@cache_page(60 * 60 * 24)
+def funder_dashboard(request):
```

Comment:
Any chance we can DRY up everything newly introduced throughout the PR, make variables a little bit more descriptive, and maybe add some comments and/or adjust some of the loop logic to make maintainability/readability easier?